### PR TITLE
added prometheus-adapter

### PR DESCRIPTION
**Pull request by lnovara**
Added prometheus-adapter in order to enable the HorizontalPodAutoscaler to auto-scale workloads using Prometheus metrics.

### CODE REVIEW

File: katalog/prometheus-adapter/apiservice.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: apiregistration.k8s.io/v1beta1
```
Comment (iknite) [inline katalog/prometheus-adapter/apiservice.yml:1]: It's a good sanity check for yaml files to have this as begining of the file. `---`
https://yaml.org/spec/1.0/#id2561718

I use **yamllint**.  Yaml syntax are not the place to be confortable debugging.

File: katalog/prometheus-adapter/config.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+rules:
```
Comment (iknite) [inline katalog/prometheus-adapter/config.yml:1]: ---

File: katalog/prometheus-adapter/config.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- seriesQuery: '{__name__=~"^container_.*",container_name!="POD",namespace!="",pod_name!=""}'
```
Comment (iknite) [inline katalog/prometheus-adapter/config.yml:2]: Also list should use two spaces before the hypen (-)

**Comment by phisco** [comment]
@iknite if you have any style tool to suggest (for vim if possible 👑 ) let's standardize all on that!
@lzecca78 I know uses something really aggressive.

**Comment by iknite** [comment]
I use yamllint. Yaml syntax are not the place to be confortable debugging.
https://github.com/dense-analysis/ale detects that is installed and shows the correct output 🚀

File: katalog/prometheus-adapter/apiservice.yml (hunk 1 above)
Comment (lzecca78) [reply katalog/prometheus-adapter/apiservice.yml:1]: totally agree @iknite, my OCD is getting worst everytime i see a yaml without `---`

**Comment by jnardiello** [comment]
@lnovara Can you please add documentation for this PR? Also, it's almost 1 year old - is it still relevant?

**Pull request closed without merging**